vietanh2810/Sorting-Visualizer-VA
# Version 1.0

## Summary

This merge request resolves issue #17

## Changes Made

- Added bars representation
- Added Merge sort and Generate new array button with its implementation

## Verification Steps

To verify that it is working correctly, follow these steps:

1. npm start
2. go to localhost:3000 to test

## Reviewers

@vietanh2810
